uaclient/config.py

```diff
@@ -729,11 +729,13 @@ def _attached_status(self) -> Dict[str, Any]:
             if not resource["available"]
         }
 
-        for ent_cls in ENTITLEMENT_CLASSES:
-            ent = ent_cls(self)
-            response["services"].append(
-                self._attached_service_status(ent, inapplicable_resources)
-            )
+        for resource in sorted(resources, key=lambda x: x["name"]):
```

Comment:
Why the resources need to be sorted ?

Reply:
We currently rely on it to show them in order when presenting the tabular output

Reply:
Sorry, just one more thing. Let's use `get` here to access the resource name. It is unlikely that we will receive a resource without that key, but we could be defensive anyway

Reply:
Hmm, I would like to keep this as-is, for:
a) we do want to raise an error in this case: a resource without name makes no sense, and no default applies if we don't find one
b) mypy complains that we use it, and we need to do some type juggling to avoid those same errors of a)

Reply:
We could just use `get("name", "")` in that way we would not break if the resource doesn't have a name and we would not break either by trying to access the entitlement on the dict

Reply:
oki, done